main.py

```diff
@@ -36,7 +36,10 @@ def _parse_env_vars() -> types_.UserInputs:
     delete_topics = os.getenv("INPUT_DELETE_TOPICS") == "true"
     dry_run = os.getenv("INPUT_DRY_RUN") == "true"
     github_access_token = os.getenv("INPUT_GITHUB_TOKEN")
-    base_branch = os.getenv("INPUT_BASE_BRANCH")
+    # Always defined since the input has a default value
+    base_tag_name = typing.cast(str, os.getenv("INPUT_BASE_TAG_NAME"))
+    # Always defined by GitHub
+    commit_sha = typing.cast(str, os.getenv("GITHUB_SHA"))
```

Comment:
I think you can use `os.environ["INPUT_BASE_TAG_NAME"]` to avoid casting.

Reply:
Good point, changed